suricata.yaml.in

```diff
@@ -753,6 +753,7 @@ dpdk:
       # - auto takes all cores
       # in IPS mode it is required to specify the number of cores and the numbers on both interfaces must match
       threads: auto
+      interrupt-mode: no # yes to switch to interrupt mode 
```

Comment:
Comment out if "no" is also the default.

Reply:
Also use `true`/`false` as that is what is used in other boolean fields in this section.

Reply:
changed